packages/terra-dialog/CHANGELOG.md

```diff
@@ -3,4 +3,7 @@ ChangeLog
 
 Unreleased
 -----------------
-* Initial stable release
+* New package scaffold.
```

Comment:
Typicaly we leave this text as `* Initial stable release` for the v1.0.0 release